package.json

```diff
@@ -251,7 +250,9 @@
       "ethereum-public-key-to-address>keccak": false,
       "parcel-bundler>@parcel/watcher>chokidar>fsevents": false,
       "ws>bufferutil": false,
-      "ws>utf-8-validate": false
+      "ws>utf-8-validate": false,
+      "gridplus-sdk>Buffer": true,
+      "gridplus-sdk": true
```

Comment:
can you expand on which scripts need to run here?

Reply:
Set this to `false`. [Commit](https://github.com/floating/frame/pull/845/commits/bbe30a3fc6f6be1671eea779df4a1aaac6bba632)